src/views/Home.jsx

```diff
@@ -26,60 +31,109 @@ export function Home({ handleListTokenState }) {
 		if (userEnteredToken.length === 0) {
-			return setMessage('Please enter a token.');
+			toast({
+				title: 'Error',
+				description: 'Please enter a token.',
+				variant: 'errorToast',
+			});
+			return;
 		}
 		streamListItems(userEnteredToken, (snapshot) => {
 			if (snapshot.empty) {
-				setMessage('List not found. Please try another token.');
+				toast({
+					title: 'Error',
+					description: 'List not found. Please try another token.',
+					variant: 'errorToast',
+				});
 			} else {
 				handleListTokenState(userEnteredToken);
 			}
 		});
 	};
 
 	return (
-		<div
-			className="Home"
-			style={{
-				display: 'flex',
-				flexDirection: 'column',
-				alignItems: 'center',
-				gap: '1rem',
-			}}
+		<Flex
+			direction="column"
+			h="100%"
+			align="center"
+			justify="space-around"
+			pt={4}
 		>
-			<p>
-				Hello from the home (<code>/</code>) page!
-			</p>
-			<button onClick={handleCreateList}>Create a new list</button>
-
-			<p>or</p>
-			<form
-				onSubmit={handleFormSubmit}
-				style={{
-					display: 'flex',
-					flexDirection: 'column',
-					alignItems: 'center',
-					gap: '1rem',
-				}}
-			>
-				<label htmlFor="join-list">Join an existing list</label>
-				<input
-					type="text"
-					id="join-list"
-					value={userEnteredToken}
-					onChange={handleChange}
-				/>
-				<button>Join</button>
-				{message && <p>{message}</p>}
-			</form>
-		</div>
+			<Box pb={4}>
+				<Button
+					bg="soon.500"
+					textColor="brand.500"
+					onClick={handleCreateList}
+					fontSize={{ base: 'md', md: 'xl' }}
+				>
+					Create a new list
+				</Button>
+			</Box>
+			<Divider borderWidth={'2px'} borderColor="text.500" />
+			<FormControl id="join-list">
+				<form onSubmit={handleFormSubmit}>
+					<Flex
+						direction="column"
+						align="center"
+						justify="center"
+						gap={2}
+						pt={4}
+					>
+						<FormLabel htmlFor="join-list" fontSize={{ base: 'md', md: 'xl' }}>
+							Join an existing list
+						</FormLabel>
+						<Box>
+							<InputGroup size="md">
+								<Input
+									bg="darkBackground.500"
+									mb={2}
+									type="text"
+									id="join-list"
+									value={userEnteredToken}
+									onChange={handleChange}
+								/>
+								<InputRightElement>
+									<IconButton
+										aria-label="Clear token"
+										icon={<X />}
+										onClick={() => setUserEnteredToken('')}
+										_hover={{ bg: 'none' }}
+										bg="none"
+										display={
+											userEnteredToken.length > 0 ? 'inline-flex' : 'none'
+										}
+									/>
+								</InputRightElement>
+							</InputGroup>
+						</Box>
+						<Button
+							type="submit"
+							bg="soon.500"
+							textColor="brand.500"
+							fontSize={{ base: 'md', md: 'xl' }}
+						>
+							Join
+						</Button>
+					</Flex>
+				</form>
+			</FormControl>
+			<Image
+				src={'./img/people.svg'}
+				w="80%"
+				alt="Shoppers at a grocery store enjoying their experience"
```

Comment:
Rather than using % for w, I think we should use a fixed px instead, since upon expanding to desktop size the gif is humongous. Or the other option could be to use breakpoints, but I think fixed width across the platform should suffice.

Reply:
This is a good idea! We set max-width to a set number of pixels so that it scales with phone size but doesn't get out of control.